src/Entity/ThematicCommunity/AdherentMembership.php

```diff
@@ -57,4 +49,27 @@ public function getPhone(): ?PhoneNumber
     {
         return $this->adherent ? $this->adherent->getPhone() : null;
     }
+
+    public function getPostAddress(): ?PostAddress
+    {
+        return $this->adherent->getPostAddressModel();
+    }
+
+    public function getJob(): ?string
+    {
+        return $this->adherent ? $this->adherent->getJob() : null;
+    }
+
+    public function getCityWithZipcode(): string
```

Comment:
it can be moved into AbstractMembership, no?

Reply:
yes, but I will have to determine which type of object I have before. Should I still move this into the abstract ?

Reply:
as seen together, added some abstract methods, and moved this one to the parent